### ISSUE TITLE
Installer: Allow the installation of NVDA to be Executed as a Different User

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#9434]
As part of my job, I often need to install NVDA on workstations. The setup of these systems is that there is a local administrator account, and Windows domain accounts - the latter of these does not have admin permissions over the system, and as a result I first need to choose to specifically run NVDA as this alternative account - which isn't always the easiest thing to do if relying on memory to find the file, and choose the proper option.

To combat this, when choosing to install NVDA would it be possible to add an option that allows the user to install NVDA under a different user account - other than the one that the current NVDA executable has been started from?
